### ISSUE TITLE
[TW-920] webdav sync problem

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#955]
_Paul Beckingham on 2011-02-07T08:23:15Z says:_ 

 Hello,

I would like to report a bug in the synchronization capabilities when
using webdav as a transport protocol. My setup involves a webdav
repository at https://server.expample/dav/task/ and a local task
directory in $HOME/.task/. After a 
   task push
only pending.data appears in https://server.expample/dav/task/, but
undo.data and completed.data are missing.

The command line to curl generated by task looks as follows:

   curl --user fcengel -T /net/home/engel/.task/pending.data \
* T /net/home/engel/.task/undo.data \
* T /net/home/engel/.task/completed.data \
   https://server.example/dav/task/

According to the curl manpage, the URL should be given after each
* T option, from man curl:
   You can specify one -T for each URL on the command line. Each -T
   + URL pair specifies what to upload and to where.
This has been verified for:
* curl 7.15.5 (x86_64-redhat-linux-gnu) libcurl/7.15.5 OpenSSL/0.9.8b zlib/1.2.3 libidn/0.6.5
* curl 7.18.2 (x86_64-pc-linux-gnu) libcurl/7.18.2 OpenSSL/0.9.8g zlib/1.2.3.3 libidn/1.8 libssh2/0.18

I am using task-1.9.3 and have compiled it myself. 

The output of task diag is here:

[0]> task diag

task 1.9.3
 Platform: Linux

Compiler
  Version: 4.1.2 20080704 (Red Hat 4.1.2-46)
     Caps: +stdc +stdc_hosted +200112 +LP64 +c1 +i4 +l8 +vp8

Libraries
  NCurses: 5.5 (20060715)
 Readline: n/a
      Lua: n/a

Build Features
    Built: Jan 20 2011 09:55:13
     Caps: +pthreads +srandom +random -uuid

Configuration
     File: /net/home/engel/.taskrc (found), 2659 bytes, mode 100640
     Data: /net/home/engel/.task (found), dir, mode 40750

External Utilities
      scp: found
    rsync: 3.0.2
     curl: 7.15.5

Tests
 UUID gen: 1000 unique UUIDs generated.

Best regards,
   Felix Engel

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2011-02-07T08:23:15Z
Modified: 2014-02-09T02:11:52Z
```

**Comment by taskwarrior** [comment]
_Johannes Schlatow on 2011-02-07T16:32:30Z says:_ 

 Already fixed in 1.9.4 (see #548).
